src/Aggregate/AggregateRoot.php

```diff
@@ -13,7 +13,7 @@ abstract class AggregateRoot
 {
     /** @var AggregateChanged[] */
     private array $uncommittedEvents = [];
-    private int $playhead = -1;
+    protected int $playhead = -1;
```

Comment:
im not quite happy with this but i dont think we can let it be as private since we need to read, set & update the value in the child class.

Reply:
at least i set it to internal

Reply:
i don't think we should start at 0 with the playhead. i find 1 more logical at the first event.

Reply:
we should discuss that in another issue